src/python/pants/goal/context.py

```diff
@@ -80,8 +80,6 @@ def __init__(self, options, run_tracker, target_roots,
     self._workspace = workspace or (ScmWorkspace(self._scm) if self._scm else None)
     self._replace_targets(target_roots)
     self._invalidation_report = invalidation_report
-    # TODO(#4769): This should not be exposed to anyone.
-    # Note that the Context created in unit tests by BaseTest uses a different codepath.
```

Comment:
This note is probably still useful...

(I think @stuhood had plans to unify them? :))

Reply:
that work should already be tracked in https://github.com/pantsbuild/pants/issues/4401 fwict